components/EventCard/EventCard.vue

```diff
@@ -1,164 +1,129 @@
 <template>
-  <div class="event-card">
-    <div class="event-content">
-      <div class="image">
-        <event-banner-image :src="bannerUrl" />
-      </div>
-      <div class="event-content-wrap">
-        <h3>
-          <nuxt-link
-            :to="{ name: 'events-eventId', params: { eventId: event.sys.id } }"
-          >
-            {{ event.fields.title }}
-          </nuxt-link>
-        </h3>
-        <div class="subtitle">
-          <div>{{ event.fields.location }}</div>
-          <!-- <div>{{ startDate }} - {{ endDate }}</div> -->
-        </div>
-        <div class="details">
-          <div class="detail">
-            {{ event.fields.summary }}
-          </div>
-          <div class="detail eventinfo">
-            {{ event.fields.eventType }} - SPARC Attendees:
-            {{ event.fields.sparcAttendees }}
-          </div>
-        </div>
-      </div>
+  <div class="upcoming-event">
+    <div class="upcoming-event__image">
+      <img :src="eventImage(event)" :alt="eventAlt(event)" />
```

Comment:
This is what I was talking about here. If some events don't have images, you can add a `v-show` to the div class to display the image only if it exists, unless there's a different behavior that the team wants.